CMSgov/design-system
# [WNMGDS-312] Update margin and padding examples

## Summary

- Add pixel values to margin and padding examples
- Update examples design to a more conventional format like found on the colors page (example, class, value) 
- Added control to render padding and margin examples
- Added styles for margin and padding rendering

## How to test

1. run the doc site with `yarn start` 
2. the visit the margin page and padding page to see updated examples with pixel values 

## Checklist




### If this is a change to code:


### If this is a change to documentation:
